scripts/build.sh

```diff
@@ -2,6 +2,7 @@
 
 set -euo pipefail
 
+GOOS="linux" go build -ldflags='-s -w' -o bin/helper github.com/paketo-buildpacks/datadog/cmd/helper
```

Comment:
could you elaborate the reason to change from main to helper?

Reply:
not changing, rather just adding this to the existing build of 'main' which is still on the next line. I'm not 100% certain why it is done this way, but this is how helpers are implemented in other projects.